src/style.css

```diff
@@ -2,0 +1,29 @@
+.stars {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.stars__row {
+  display: flex;
+  gap: 5px;
+}
+
+.stars__star {
+  width: 30px;
+  height: 30px;
+  display: inline-block;
+  background-size: cover;
+  background-image: url('../src/images/star.svg');
```

Comment:
Consider using a more descriptive name for the image file to reflect its purpose or appearance, such as 'star-icon.svg' or 'star-outline.svg'. This can improve maintainability and readability of the code.